src/server.js

```diff
@@ -477,6 +488,14 @@ function getClientConfiguration() {
 	return config;
 }
 
+function getServerConfiguration() {
+	const config = Helper.config;
+
+	config.customCss = packages.getCustomCss();
```

Comment:
You're assigning this to `Helper.config` object.

Reply:
Yuuuup, good point. I shall clone it.